packages/dart_frog_test/lib/src/dart_frog_test_context.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:dart_frog/dart_frog.dart';
+import 'package:mocktail/mocktail.dart';
+
+class _MockRequestContext extends Mock implements RequestContext {}
+
+/// {@template dart_frog_test_context}
+/// A context used to test Dart Frog handlers.
+/// {@endtemplate}
+class DartFrogTestContext {
+  /// {@macro dart_frog_test_context}
+  DartFrogTestContext({
+    required this.path,
+    this.basePath = 'https://test.com',
+    this.method = HttpMethod.get,
+    this.headers,
+    this.body,
+  });
+
+  /// The route path.
```

Comment:
Should we [consider adding examples](https://dart.dev/effective-dart/documentation#consider-including-code-samples-in-doc-comments) to some of these members documentation (maybe also to some matchers)? So when a developer hovers in their IDEs it is immediately obvious to them the format and what has to be provided.

Reply:
Hum, I feel that these are quite obvious, I don't really feel the need, unless you see something that I might be forgetting?